CHANGELOG-v3.adoc

```diff
@@ -5,6 +5,20 @@ For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim
 *Important:* If you plan to skip versions when you upgrade, ensure that you read the version-specific upgrade notes for each intermediate version. You may be required to perform manual actions as part of the upgrade.
 
 
+== APIM - 3.10.21 (2022-09-16)
+
+=== Improvements
+
+*_Gateway_*
+
+- Worker pool size of the HTTP handlers in Bridge mode is now configurable. Configurable property is `services.bridge.http.workerPoolSize`, default value is `20`, and it has to be set on the Gateway Bridge Server side.
```

Comment:
```suggestion
- We have added a new property that enables you to configure the worker pool size of the HTTP handlers in Bridge mode. To do so, set the desired size using the `services.bridge.http.workerPoolSize` property (the default value is `20`) and ensure that it is set on the Gateway Bridge Server side.
```

Reply:
@gaetanmaisse As per the rewrite above I am saying this is a new property - but is it new? Just making sure:)

Reply:
Yes, it's brand 🆕!
Thanks for the feedback, I update it right now